citra_libretro.info

```diff
@@ -1,5 +1,5 @@
 # Software Information
-display_name = "Nintendo - 3DS (Citra)"
+display_name = "Nintendo - 3DS/new3DS (Citra)"
```

Comment:
New 3DS?

Reply:
I guess they're treating it as its own thing, like DSi, which has a certain logic to it, assuming there are n3DS-specific games (the SNES VC is one, I guess).